SEMANTIC-VERSIONING.md

```diff
@@ -0,0 +1,137 @@
+#Semantic Versioning
+
+When a new version of Tilezen is released, developers should be able to tell from the version increment how much effort it will take them to integrate the new tiles with their map. We use semantic versioning to communicate this.
+
+###What is Semantic Versioning?
+
+Semantic versioning (or [SemVer](http://semver.org/)) is a formalized way of making promises with an X.Y.Z version indicator.
+
+#### Version components
+
+- `MAJOR`.`MINOR`.`PATCH`, example: `1.0.0` (default)
+- `MAJOR`.`MINOR`.`PATCH`-`POSTFIX`, example: `1.0.0-pre1` (optional)
+
+#### Version parts:
+
+1. **MAJOR** version **X** for incompatible API changes.
+2. **MINOR** version **Y** when adding functionality in a backwards-compatible manner, and
+3. **PATCH** version **Z** when fixing backwards-compatible bugs
+4. **pre-release** version **-POSTFIX** when releasing developer previews
+
+**NOTE:** The above applies _after_ software has reached as version `1.0.0`, no promises are made for earlier versions.
+
+#### Developer level of effort:
+
+- Major version X: **high** – significant integration challenges, read the changelog closely
+- Minor version Y: **low** – some integration challenges, read the changelog
+- Bug fixes Z: **none** – simply use the new tiles, ignore the changelog
+- Pre-release POSTFIX: **low to high** – some to significant integration challenges, read the changelog
+
+## Versioning the Tilezen API
+
+Proposed that upon our `1.0.0` release Tilezen makes the following promises.
+
```

Comment:
Added in 6ec85ead90225bb710d61122a25068f5f378cce4.

It feels like this needs a little bit more work, but it's a good straw man. Specifically: the descriptions added are pickups of WOF placetype roles, and match fairly well with what's in the Layers documentation now. But it feels like it needs slightly more finesse around which promises are made.